src/JustSaying/AwsTools/MessageHandling/MessageDispatcher.cs

```diff
@@ -145,7 +145,9 @@ private async Task<bool> CallMessageHandler(Message message)
 
             if (handler == null)
             {
-                return true;
+                _logger.LogError("Handler for message of type '{MessageTypeName}' not found in handler map. Returning message to queue.", message.GetType().FullName);
+                // Return false so that we give other queue subscribers a chance to handle the message
```

Comment:
Very minor nit; I think the phrasing should indicate that dispatching has failed, rather than implying this is a supported pattern.
So maybe:
```csharp
_logger.LogError("Failed to dispatch. Handler for message of type '{MessageTypeName}' not found in handler map.", message.GetType().FullName);
```
and no comment.

Reply:
Done.